rn/Teacher/src/modules/dashboard/account-notification-reducer.js

```diff
@@ -76,4 +80,32 @@ export const accountNotifications = handleActions({
       }
     },
   }),
+  [AccountNotificationActions.refreshLiveConferences.toString()]: handleAsync({
+    pending: (state) => ({
+      ...state,
+      liveConferencesPending: state.pending + 1,
+      liveConferencesError: '',
+    }),
+    resolved: (state, { result: { data } }) => ({
+      ...state,
+      liveConferencesPending: Math.max(0, state.pending - 1),
+      liveConferences: data,
```

Comment:
This needs to be `data.conferences`. Tried this Adrian's changes locally and the conferences list is nested inside a top-level `conferences` key.

```
{"conferences":[]}
```